libsupport/test/cache.cpp

```diff
@@ -7,45 +7,11 @@
 #include "katana/Logging.h"
 #include "katana/Random.h"
 
-enum class NodeEdge { kNode, kEdge };
-
-// This code is replicated from libtsuba::PropertyCache.h to allow testing of
-// libsupport::Cache.h (libtsuba depends on libsupport, not the other way around)
```

Comment:
Do you also have to remove `PropertyCacheKey` in `libtsuba::PropertyCache.h`?

Reply:
Unless this is still being used. I'm not too sure.

Reply:
There is no more` libtsuba::PropertyCache.h`.

Reply:
Oops! Skipped over the file that was completely deleted :)